src/sentry/stacktraces/processing.py

```diff
@@ -223,6 +230,15 @@ def _append_stacktrace(stacktrace: Any, container: Any, is_exception: bool = Fal
     return rv
 
 
+def extract_structure_from_json_structure(data: Any, *path: str) -> list[Any]:
+    """Helper method to extract a data structure from a JSON object"""
+    rv = []
+    # filter=True filters out any None element
+    for item in get_path(data, path, filter=True, default=()):
+        rv.append(item)
+    return rv
```

Comment:
Two suggestions here:
- This only gets lists, so IMHO we should say so in the function name.
- Having `filter_nones` in the signature makes it a little more obvious that that's happening, which you'd otherwise have to look at the implementation to see.

```suggestion
def extract_list_from_json(data: Any, *path: str, filter_nones=True) -> list[Any]:
    """Helper method to extract a list from a JSON object"""
    rv = []
    # filter=True filters out any None element
    filter = True if filter_nones else None
    for item in get_path(data, path, filter=filter, default=()):
        rv.append(item)
    return rv
```